### PR TITLE
added shortcut creation into setup.py

### PR DESCRIPTION
**Pull request by Rw552**
Got that working, made the new .msi

### CODE REVIEW

**Comment by Rw552** [comment]
wait hold on
still busted lol

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 setup(
     name="RAstral Rampart",
-    version="1.0.0",
+    version="1.0.1",
```
Comment (chwerth) [inline setup.py]:
```suggestion
    version="1.1.0",
```
minor but I think this is the right versioning for this sort of change

https://semver.org/#summary

**Comment by Rw552** [comment]
So the .exe still runs, but the shortcut gives me this @chwerth 
image

**Comment by chwerth** [comment]
Looks like a common problem @Rw552 

https://stackoverflow.com/questions/24195311/how-to-set-shortcut-working-directory-in-cx-freeze-msi-bundle

Apparently, you need to set the working directory to be the installation directory

**Comment by Rw552** [comment]
How would I go about doing that? modifying the installation of cxfreeze?

**Comment by chwerth** [comment]
Is it possible to set up a shortcut table like the second guy answered?

https://stackoverflow.com/a/52618332

**Comment by Rw552** [comment]
yeah I'm working on that. I'm assuming TARGETDIR should be replaced with whatever directory i want the program to be installed in?

**Comment by chwerth** [comment]
Just from glancing at documentation, I think its a built-in term?

https://docs.microsoft.com/en-us/windows/win32/msi/targetdir

**Comment by Rw552** [comment]
I can't seem to get it to work

**Comment by chwerth** [comment]
Aw. Well that's ok we could just leave this alone for now.

**Comment by Rw552** [comment]
ok

**Comment by chwerth** [comment]
Closing until we can figure this out

**Pull request closed without merging**